examples/spark2/src/main/scala/org/apache/carbondata/examples/CarbonPartitionExample.scala

```diff
@@ -101,17 +126,40 @@ object CarbonPartitionExample {
     spark.sql("""
        | CREATE TABLE IF NOT EXISTS t5
        | (
+       | id Int,
        | vin String,
        | logdate Timestamp,
        | phonenumber Long,
-       | area String
+       | area String,
+       | salary Int
        |)
        | PARTITIONED BY (country String)
        | STORED BY 'carbondata'
        | TBLPROPERTIES('PARTITION_TYPE'='LIST',
-       | 'LIST_INFO'='(China,United States),UK ,japan,(Canada,Russia), South Korea ')
+       | 'LIST_INFO'='(China, US),UK ,Japan,(Canada,Russia, Good, NotGood), Korea ')
```

Comment:
Hi @chenerlu , here in DDL statement, it's designed to leave no space to mock real situation which could happen in customer writing.